project-management/1-1-meetings/Anne-Lee-Steele/2024-checkins-malvika.md

```diff
@@ -0,0 +1,185 @@
+## 1:1 meetings
+
+# 23-04-2024
+- Preparation for community forum
+- Community health - presenting tomorrow at RCM team meeting
+- Focus for this week: data protection policy, newsletter, comms for book dash, accessibility guide draft
+- While MS is on AL: what would you like me to prioritise?
+	- Filing for some AL in May - family is visiting!
+- Progress report
+	- Annual Review 2023 - edited documentation as needed
+	- Documentation post - will add tomorrow
```

Comment:
Community Management document - create an issue to organise them for review by delivery team